src/components/app-bar/icons/search-icon.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16" fill="none">
+  <path d="M14.4733 13.5266L10.8733 9.93331C11.6055 9.00212 12.0024 7.85126 11.9999 6.66665C11.9999 3.72113 9.6121 1.33331 6.66659 1.33331C3.72107 1.33331 1.33325 3.72113 1.33325 6.66665C1.33325 9.61217 3.72107 12 6.66659 12C7.8512 12.0025 9.00206 11.6056 9.93325 10.8733L13.5266 14.4733C13.6518 14.5995 13.8222 14.6705 13.9999 14.6705C14.1777 14.6705 14.3481 14.5995 14.4733 14.4733C14.5995 14.3481 14.6705 14.1777 14.6705 14C14.6705 13.8222 14.5995 13.6518 14.4733 13.5266ZM2.66659 6.66665C2.66659 4.45751 4.45745 2.66665 6.66659 2.66665C8.87572 2.66665 10.6666 4.45751 10.6666 6.66665C10.6666 8.87579 8.87572 10.6666 6.66659 10.6666C4.45745 10.6666 2.66659 8.87579 2.66659 6.66665Z" fill="#9FAEBC"/>
```

Comment:
**suggestion:** Not a big deal in this instance, but I think its usually good to change this to be `fill="currentColor"` so that the SVG inherits its parent's colour.